migrations/utils/email.go

```diff
@@ -0,0 +1,60 @@
+package utils
+
+import (
+	"encoding/base64"
+	"fmt"
+	"log/slog"
+	"net/smtp"
+	"os"
+	"runtime/debug"
+	"strings"
+)
+
+func sendEmail(subject, body string, to []string) {
+	// server and from/to
+	host := "aspmx.l.google.com"
+	port := "25"
+	from := "oda-noreply@met.no"
+
+	// add stuff to headers and make the message body
+	header := make(map[string]string)
+	header["From"] = from
+	header["To"] = strings.Join(to, ",")
+	header["Subject"] = subject
+	header["MIME-Version"] = "1.0"
+	header["Content-Type"] = "text/plain; charset=\"utf-8\""
+	header["Content-Transfer-Encoding"] = "base64"
+	message := ""
+	for k, v := range header {
+		message += fmt.Sprintf("%s: %s\r\n", k, v)
+	}
+
+	body = body + "\n\n" + fmt.Sprintf("Ran with the following command:\n%s", strings.Join(os.Args, " "))
+	message += "\r\n" + base64.StdEncoding.EncodeToString([]byte(body))
+
+	// send the email
+	err := smtp.SendMail(host+":"+port, nil, from, to, []byte(message))
```

Comment:
Does this not require authentication?

Reply:
I made a note recently that my original code probably needs updating because of announced changes in DMARC settings for email. These are the current guidelines:
https://it.pages.met.no/infra/brukerdokumentasjon/smtp.html

Reply:
To be honest, I haven't really tested the email code, maybe I should simply remove it :sweat_smile: